psydac/api/ast/parser.py

```diff
@@ -486,14 +486,14 @@ def _visit_DefNode(self, expr, **kwargs):
         args = [tuple(arg.values())[0] if isinstance(arg, dict) else arg for arg in args]
         arguments = flatten(args) + mats
 
-        if constants:
-            arguments += [self._visit(i, **kwargs) for i in constants]
-
         if f_args:
             f_args     = [self._visit(i, **kwargs) for i in f_args]
             f_args     = [tuple(arg.values())[0] if isinstance(arg, dict) else arg for arg in f_args]
             arguments += flatten(f_args)
 
+        if constants:
+            arguments += [self._visit(i, **kwargs) for i in constants]
+
```

Comment:
So you changed the order of the arguments? I think that it should be documented somewhere, maybe in the docstring of this method? Suggestions are welcome

Reply:
added in [a31334c](https://github.com/pyccel/psydac/pull/248/commits/a31334cb664a6ebc9abd1bd4411bdae7d7609ec2)